StreetPatch.Data/Entities/DTO/SignUpData.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StreetPatch.Data.Entities
+{
+    public class SignUpData
```

Comment:
Please follow coding conventions - if this is a DTO, it should be named respectively - `SignUpDto`, for example.